crates/vm/src/system/public_values/core.rs

```diff
@@ -48,6 +48,10 @@ impl<F: Field> BaseAirWithPublicValues<F> for PublicValuesCoreAir {
     fn num_public_values(&self) -> usize {
         self.num_custom_pvs
     }
+
+    fn columns(&self) -> Vec<String> {
+        todo!()
```

Comment:
Not sure about this particular case

Reply:
In that case, we would have to reintroduce the macro for Vec<T> that we seemed to be able to eliminate here: https://github.com/powdr-labs/openvm/pull/1#discussion_r1997708135.

But I understand that this would be solved if we just modify Air, so I'll put it on hold.